libs/platform/src/lib/components/menu-button/menu-button.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, ChangeDetectionStrategy, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { InputSize } from './../form/form-control';
+import { ButtonType } from '../button/button.component';
+import { ButtonOptions } from '../button/button.component';
+
+@Component({
+    selector: 'fdp-menu-button',
+    templateUrl: './menu-button.component.html',
+    styleUrls: ['./menu-button.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
Again, since we are using onPush change detection strategy, you want to take under control change detection. There are cases, that might not update as you are not handling it. 

You want to probably inject ChangeDectionRef. Also what about RTL?  Do we need to insert RTL service or it is handled by core template ?

Reply:
can you show me a example on how to use RTL service, i  did not find in any component developed till now. i will remove ChangeDetectionStrategy.OnPush, hope this solves the problem and i don't have to include 'ChangeDectionRef'.

Reply:
i don't see RTL handled in core template as well

Reply:
RTL is working. see screenshot.
<img width="1451" alt="RTL" src="https://user-images.githubusercontent.com/57661487/77645520-d780cf80-6f88-11ea-8671-b9029eeda408.png">
<img width="1442" alt="LTR" src="https://user-images.githubusercontent.com/57661487/77645534-dc458380-6f88-11ea-8416-6b3038b25e1f.png">